docs/index.rst

```diff
@@ -100,5 +100,6 @@ Horizon 2020 European Research Infrastructure project (676541).
    config_reference
    howto/initialize_cells
    minimal_example
+   pythreejs-example
```

Comment:
Would it make more sense to add the example under [examples section](https://thebe.readthedocs.io/en/latest/index.html#examples) of the docs?

Reply:
Yes, but none of those are in the Sphinx docs. This table of contents `toc` typically shows all the rst files. The examples in `examples/` need to be moved into the Sphinx docs and then we could have an examples section in the table of contents.

Reply:
maybe creating an examples directory in the sphinx docs is appropriate for these? It might make sense to move all of the existing examples in there

Reply:
Can do.

Reply:
+1 to moving all of our example pages to `examples/` and adding a section in the docs for them specifically

I don't know that we need a separate *index* file for `examples`, we can just link them from the front page like:

```
.. toctree::
   :caption: Examples

   examples/ex1
   examples/ex2
   etc
```

does that make sense?